test/babashka/fs_test.clj

```diff
@@ -835,3 +835,10 @@
           file-in-dir (fs/create-temp-file {:dir dir})]
       (is (= (str (fs/owner dir)) (str (fs/owner file-in-dir)))))))
 
+(deftest issue-135-test
+  (def u (java.net.URI/create (str "jar:file:" (fs/cwd) "/test-resources/bencode-1.1.0.jar")))
```

Comment:
@Sohalt You probably need to call `(fs/unixify (fs/cwd))` here to make a proper backslashed jar entry on Windows

Reply:
Thank you! (you're soo fast 🚀)